test/unit/models/users_model.test.js

```diff
@@ -15,26 +23,172 @@ describe('create new users', () => {
     client = await MongoClient.connect(url);
     db = await client.db(dbName);
     users = db.collection('users');
+
+    // create user
+    const email = 'user@email.com';
+    const name = 'user';
+    const password = '1234';
+
+    await storeUser(users, name, email, password);
+
+    // like recipes
+    await likesRecipes(users, email, [151266, 237491, 231170, 26692, 139012, 20669, 33474, 246255]);
+
+    // add allergies
+    await addAllergies(users, email, ['shrimp']);
   });
 
   afterAll(async () => {
     await db.clean();
   });
 
-  it('user is created successfully', async () => {
-    const email = 'user@email.com';
-    const name = 'user';
-    const password = '1234';
+  describe('selectAllUsers unit tests', () => {
+    it('selectAllUsers gets all user', async () => {
+      selectAllUsers(users, (err, res) => {
+        expect(err).toBeNull();
+        expect(res).toEqual(
+          expect.arrayContaining([
+            expect.objectContaining({
+              email: 'user@email.com',
+            }),
+          ]),
```

Comment:
not loving this layered expect, can we replace this with expect(res).toContain(expect.objectContaining({ email: 'user@email.com'})

Reply:
I tried doing that and ran into error. I googled it and found issue with JEST on github: https://github.com/facebook/jest/issues/3954